### ISSUE TITLE
Vibrant deck with night mode?

### ISSUE THREAD

**Issue by piebaker** [libvibrant/vibrantDeck#10]
Is vibrant deck and night mode compatible together? I've noticed that when enabling vibrant deck night mode disables. Would it be possible to have them both enabled together? Perhaps just taking the night mode value and overlaying it with vibrant deck

**Comment by Scrumplex** [comment]
Right now I don't see an easy way of achieving this. Maybe I can implement a check that disables changing Gamma settings when Night Mode is on

**Comment by TheKerevas** [comment]
Night mode works with vibrantDeck active. Only when I restart Steam Deck do I have to reapply it

**Comment by brandon-hurler** [comment]
@TheKerevas that's not fully true. The two conflict with each other depending on interactions. 

If you have Vibrant Deck enabled and toggle Night Mode on manually, it works as expected and Night Mode activates. 

If you restart with Night Mode enabled and Vibrant Deck enabled, Vibrant Deck will apply after Night Mode during startup, functionally disabling it until you toggle Night Mode on/off again manually. 

If you have Night Mode enabled and view the Vibrant Deck settings, Vibrant Deck will override Night Mode (likely because it is allowing you to preview the real-color settings), but if you navigate away, it does not re-enable Night Mode functionality again. 

It would be great to have automated checks that detect if Night Mode is currently set to active and adjusts accordingly.

**Comment by szkottipipen** [comment]
@brandon-hurler has very right! With fully description.
+1

**Comment by jasperkm** [comment]
@Scrumplex  maybe you can set a schedule to disable, so you can manually set vibrantdeck it to disable 1 min before night more and enable 1 min after?

**Comment by szkottipipen** [comment]
@jasperkm Yes, we can resolve with manual turn ON/OFF, juts it can be so comfortable, if the vibrantdeck check that night mode is on, and dont turn ON itself in this case.

**Comment by brandon-hurler** [comment]
As of 1.3.1, it looks like there's even more conflicts than in 1.3.0. Now if I have night mode on and go to launch a game, vibrant deck will functionally disable night mode until I toggle it off and on again. I had to roll back to 1.3.0 to avoid this.
